docs/sheets.md

```diff
@@ -0,0 +1,48 @@
+# Manage metadata as sheets
+
+Lamin offers a spreadsheet-like experience for editing metadata, particularly for biological samples used in downstream pipelines and analysis. With our enterprise plan, you can access this functionality at https://lamin.ai/&lt;instance-slug&gt;/records (see https://lamin.ai/laminlabs/lamindata/records for examples).
```

Comment:
LaminHub offers a spreadsheet-like experience for managing metadata based on LaminDB's `Record` registry.

In contrast to the pre-defined SQL-based registries like `Artifact`, `Transform`, `bionty.Gene`, etc., sheets allow adding columns flexibly.